src/object-status.scss

```diff
@@ -291,6 +291,8 @@ $inverted-color-accents: (
 
   // CLICKABLE OBJECT STATUS
   &--link {
+    @include fd-fiori-focus-within(0);
```

Comment:
Is there any reason of using focus within, instead of basic focus?

Reply:
Could you also add `0.0625rem`, instead of 0? Just to follow focus visual concept from fiori 3

Reply:
forgot to change,
@JKMarkowski  thanks
changed to fd-fiori-focus

Reply:
@JKMarkowski fixed